packages/terra-date-input/src/DateInput.jsx

```diff
@@ -654,20 +662,33 @@ class DateInput extends React.Component {
       dateValue = `${year}-${month}-${day}`;
     }
 
+    this.formatDescriptionId = (help === undefined) ? `terra-date-picker-description-format-${uuidv4()}` : '';
+
+    const format = (DateInputUtil.computedDisplayFormat(this.props.displayFormat, this.props.intl.locale) === 'month-day-year')
+      ? `(${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`
+      : `(${this.props.intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`;
+
     return (
-      <div
-        {...customProps}
-        className={dateInputClassNames}
-        ref={(element) => { this.dateInputContainer.current = element; if (refCallback) { refCallback(element); } }}
-      >
-        <input
-          // Create a hidden input for storing the name and value attributes to use when submitting the form.
-          // The data stored in the value attribute will be the visible date in the date input but formatted in YYYY-MM-DD format.
-          type="hidden"
-          name={name}
-          value={dateValue}
-        />
-        {this.formattedRender()}
+      <div>
```

Comment:
Can we remove this div?

Reply:
Removed it here https://github.com/cerner/terra-framework/pull/1230/commits/1a0a9fb157c78a81826af573dd7447d392299198.